src/Core/src/Platform/Android/TextAlignmentExtensions.cs

```diff
@@ -40,6 +41,13 @@ public static void UpdateTextAlignment(this EditText view, TextAlignment horizon
 				view.TextAlignment = horizontal.ToTextAlignment();
 				view.Gravity = vertical.ToVerticalGravityFlags();
 			}
+
+			if (OperatingSystem.IsAndroidVersionAtLeast(26))
```

Comment:
This method (`public static void UpdateTextAlignment(this EditText view, TextAlignment horizontal, TextAlignment vertical)`) is not being used inside MAUI and I would evaluate removing it from PublicAPI.

Besides that the goal here is to support text alignment on `Label` right?
If so, this code should be moved here: https://github.com/dotnet/maui/blob/2815ad956f06f165fc5e59ad064468a0c9c346ef/src/Core/src/Platform/Android/TextViewExtensions.cs#L54

If you want to also support `Entry` text alignment, you can use the above method: `internal static void UpdateHorizontalAlignment(this EditText view, TextAlignment alignment, AGravityFlags orMask = AGravityFlags.NoGravity)`

Reply:
Yea, I tested moving this into EditText as well but it didn't seem to do anything to EditText unfortunately :-(

Reply:
alright marked this one obsolete and removed non functional code